akka-stream/src/main/scala/akka/stream/impl/Sinks.scala

```diff
@@ -53,6 +51,10 @@ private[akka] abstract class SinkModule[-In, Mat](val shape: SinkShape[In]) exte
     if ((thatN eq null) || thisN == thatN) shape
     else shape.copy(in = Inlet(thatN + ".in"))
   }
+
+  protected def label: String
+  final override def toString: String = f"$label [${System.identityHashCode(this)}%08x]"
```

Comment:
The inflation bit was a JRockit trick that happened if the object was moved during GC after identity hash code. AFAIK using identity hash code on HotSPot will only prevent biased locking from being used. Which could be bad enough depending on the use case.